tests/FlightAsyncTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests;
+
+use Flight;
+use flight\Engine;
+use PHPUnit\Framework\TestCase;
+
+class FlightAsyncTest extends TestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        Flight::setEngine(new Engine());
```

Comment:
This line and method is unnecessary !!

Reply:
Without this line, other tests from other files (yes cause it's static) would blow up these tests. I wanted to make sure that this test class started with a clean slate.